sybil/integration/pytest.py

```diff
@@ -128,11 +128,13 @@ def teardown(self) -> None:
             self.sybil.teardown(self.document.namespace)
 
 
-def pytest_integration(*sybils: 'Sybil'):
+def pytest_integration(*sybils: 'Sybil') -> Callable[[Path, Collector], Optional[SybilFile]]:
 
-    def pytest_collect_file(file_path: Path, parent: Collector):
+    def pytest_collect_file(file_path: Path, parent: Collector) -> Optional[SybilFile]:
         for sybil in sybils:
             if sybil.should_parse(file_path):
-                return SybilFile.from_parent(parent, path=file_path, sybil=sybil)
+                result: SybilFile = SybilFile.from_parent(parent, path=file_path, sybil=sybil)
```

Comment:
We have to type hint this because `SybilFile.from_parent` is inherited from `pytest` and `pytest` does not type hint the return value of `from_parent`.